src/pages/home/_trade-types.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Carousel, Header, QueryImage, Text } from 'components/elements'
+import { localize, Localize, LocalizedLink } from 'components/localization'
+import { Flex, SectionContainer } from 'components/containers'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+import device from 'themes/device.js'
+import Arrow from 'images/svg/trade-types/arrow-right.svg'
+
+type TradeTypesProps = {
+    image_url: string
+    image_alt: string
+    header: any
+    desc: any
```

Comment:
Can we use ReactElement for all these types? Considering all of them should be localized